docs/UserGuide.md

```diff
@@ -89,9 +89,10 @@ Examples:
 
 ### Listing all persons : `list`
 
-Shows a list of all persons in the address book.
+Lists the existing contacts based on the group. 
+If the group name is unspecified, then list all the existing contacts.
```

Comment:
I had already updated the this part for 'list' feature, but it seems that you haven't pull from the team repo (not the one you forked to your repo) master branch. I suggest that you pull it and resolve the conflict, else I think you will have the merge conflict later when you try to merge your PR.